Microsoft.Azure.Cosmos/src/CosmosClient.cs

```diff
@@ -257,6 +258,154 @@ protected CosmosClient()
                 clientOptions);
         }
 
+        /// <summary>
+        /// Creates a new CosmosClient with the account endpoint URI string and TokenCredential.
+        /// In addition to that it initializes the client with containers provided i.e The SDK warms up the caches and 
+        /// connections before the first call to the service is made. Use this to obtain lower latency while startup of your application.
+        /// CosmosClient is thread-safe. Its recommended to maintain a single instance of CosmosClient per lifetime 
+        /// of the application which enables efficient connection management and performance. Please refer to the
+        /// <see href="https://docs.microsoft.com/azure/cosmos-db/performance-tips">performance guide</see>.
+        /// </summary>
+        /// <param name="accountEndpoint">The cosmos service endpoint to use</param>
+        /// <param name="authKeyOrResourceToken">The cosmos account key or resource token to use to create the client.</param>
+        /// <param name="containers">(Optional) Containers to be initialized</param>
```

Comment:
```suggestion
        /// <param name="containers">Containers to be initialized identified by it's database name and container name.</param>
```
It is not optional based on the API, right?